src/main/java/seedu/souschef/logic/commands/ListCommand.java

```diff
@@ -25,6 +33,17 @@ public ListCommand(Model model) {
     @Override
     public CommandResult execute(History history) {
         requireNonNull(model);
+        Context context = history.getContext();
+        if (context.equals(INGREDIENT)) {
+            model.sort(new IngredientDateComparator());
+        } else if (context.equals(CROSS)) {
+            model.updateFilteredList(Model.PREDICATE_SHOW_ALL_CROSSRECIPES);
```

Comment:
Possible to pass in a different predicate based on context rather than not using the predicate passed in.

Reply:
I think maybe replacing that one with predicate_show_all would also work. But not sure.